sympy/utilities/lambdify.py

```diff
@@ -847,7 +856,13 @@ def _lambdifygenerated(x):
         funcprinter = _TensorflowEvaluatorPrinter(printer, dummify) # type: _EvaluatorPrinter
     else:
         funcprinter = _EvaluatorPrinter(printer, dummify)
-    funcstr = funcprinter.doprint(funcname, args, expr)
+
+    if cse != False:
+        from sympy.simplify.cse_main import cse, cse_minimize_memory
+        cses, _expr = cse(expr, as_list=False, postprocess=cse_minimize_memory)
```

Comment:
I thought the purpose of allowing a callable for the lambdify cse kwarg was to allow for custom cse functions. I'd prefer that the default cse=True only calls the standard cse with no customization.

Reply:
I concur, the doc-string even says the user is allowed to pass such a function.